src/webpack.js

```diff
@@ -53,32 +67,48 @@ const getDappMetaData = () => {
 };
 
 /**
- *  @desc Get data from webpack build bundle file and combine to metadata object
- *  @param {function} callback return data with format {result:'',error:''}
+ * @desc Watching and Streaming webpack build process
+ * @param {bool} devMode return true when arg is --dev
+ * @param {function} callback return data
  */
-const getBuildObjectFromBundle = callback => {
-  const dAppMetaData = getDappMetaData();
-  const filePath = path.join(process.cwd(), 'dist/index.js');
-  fs.readFile(filePath, { encoding: 'utf-8' }, (err, data) => {
-    if (!err) {
-      callback({
-        result: {
-          ...dAppMetaData,
-          code: data,
-        },
-      });
-    } else {
-      callback({ error: err });
+const watchAndStreamData = (devMode, callback) => {
+  watchChanges(devMode, content => {
+    if (callback) {
+      //callback return data
+      callback({ content });
     }
   });
 };
 
+/**
+ * @desc Write bundle file
+ * @param {bool} devMode return true when arg is --dev
+ */
+const writeBundleFile = devMode => {
```

Comment:
I think I noticed that in the last review. this function is only supposed to write a file based on the name. not to read as well.

Reply:
changed to watchAndWriteBundleFile